chart/config/releases.yaml

```diff
@@ -42,7 +42,7 @@ releases:
   database:
     # database (previously known as pxc) is not a BOSH release.
     image:
-      repository: docker.io/cfcontainerization/pxc
+      repository: ghcr.io/cloudfoundry-incubator/pxc
```

Comment:
Is this the correct way to change the repository of this image? or can one drop the repository key and use `releases.$defaults`?

Reply:
Yes, because the helm template doesn't respect `releases.$defaults`:
https://github.com/cloudfoundry-incubator/kubecf/blob/a336ea4b70b9c940a0c82d2f8d0abc44b4807937/chart/templates/database/db-statefulset.yaml#L49-L53